### DISCUSSION TITLE
分类标签没有内容，是需要在文章上添加吗？

### DISCUSSION THREAD

**Discussion by happyhrw** [MaLuns/hexo-theme-async#39] [Q&A 疑难杂症]
![image](https://user-images.githubusercontent.com/100116732/201257799-5f5f7179-fdb4-4f63-8a10-57c0d86ae1c0.png)

**Comment by MaLuns** [comment]
需要你在文章设置先

**Reply by happyhrw** [reply]
我在文章头部写了，报错
![image](https://user-images.githubusercontent.com/100116732/201926726-e644c2d4-1491-4fbd-8b4f-5992dae4fc9d.png)
